server.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import socket
+import struct
+import sys
+import threading
+THE_UMOUNT_OF_WRONG_CONECTOIN = 5
+THE_UMOUNT_OF_BITES_IN_NUMER_OF_LEN = 4
```

Comment:
Fix spelling (amount, bytes, connection).
Also, there is no need for "the_" in constant or variable names, it doesn't really add information.